installer/sharch_body.sh

```diff
@@ -25,13 +25,20 @@ fi
 
 echo " OK."
 
+image_size=$(du "$0" | awk '{print $1}')
 # Untar and launch install script in a tmpfs
 cur_wd=$(pwd)
 export cur_wd
 archive_path=$(realpath "$0")
 tmp_dir=$(mktemp -d)
 if [ "$(id -u)" = "0" ] ; then
     mount -t tmpfs tmpfs-installer $tmp_dir || exit 1
+    mount_size=$(df $tmp_dir | tail -1 | tr -s ' ' | cut -d' ' -f4)
+    if [ "$mount_size" -lt "$((image_size*3))" ]; then
+        #align mount_size to next power of two
+        mount_size=$(echo $((image_size*3)) | awk '{ print 2^int(((log($1)/log(2))+ 1)) }')
+        mount -o remount,size="${mount_size}K" -t tmpfs tmpfs-installer $tmp_dir || exit 1
```

Comment:
Instead K, use G, so that calculation wud be easier

Reply:
chaging mount_size to G is enough right? after mounting size is aligned .below is the mout_size in K when 4G is mounted

tmpfs-installer   4194304       0   4194304   0% /tmp/tmp.vHn5rFZYE1